Package.swift

```diff
@@ -22,7 +22,8 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-nio.git", from: "2.27.0"),
-        .package(url: "https://github.com/apple/swift-nio-ssl.git", from: "2.12.0"),
+        .package(url: "https://github.com/apple/swift-nio-ssl.git", .branch("main")),
+//        .package(url: "https://github.com/apple/swift-nio-ssl.git", from: "2.12.0"),
```

Comment:
once NIOSSL is released, we need to change this back (and up the version number). Just as a reminder.

Reply:
@madsodgaard we got them release https://github.com/apple/swift-nio-ssl/releases/tag/2.13.0